src/lib/vault/dataStructures/exceptions.ts

```diff
@@ -0,0 +1,30 @@
+// Shared
+
+export class FormSubmissionNotFoundException extends Error {
```

Comment:
This submission is in the shared section as it will be used by the /problem API path